docs/DeveloperGuide.md

```diff
@@ -16,6 +16,33 @@ When the user launches the app, the main program will initialize a `TopicsStorag
 
 ![TopicsStorage load](./images/TopicsStorage_load.png)
 
+### Design of the Quiz system
+
+![QuizQuestionsManager_Class_Diagram](./images/QuizQuestionsManager.png)
+
+The Class Diagram given above explains the high-level design of the Quiz system in E-Duke-8. Given below is a quick overview of each component.
```

Comment:
Would it be better if we have a few words to introduce the diagram first?

Reply:
Alright! Thanks! I will change accordingly!